buildpacks/nodejs-engine/src/install_node.rs

```diff
@@ -23,7 +23,7 @@ use crate::{NodeJsEngineBuildpack, NodeJsEngineBuildpackError};
 
 pub(crate) fn install_node(
     context: &BuildContext<NodeJsEngineBuildpack>,
-    distribution_artifact: &Artifact<Version, Sha256>,
+    distribution_artifact: &Artifact<Version, Sha256, Option<()>>,
```

Comment:
`Option<()>` is for the metadata type on the artifact, right? Is it possible to use `()` directly since we don't (yet) support any metadata on the Node.js artifacts?

Reply:
I had a similar question :-) See this discussion/PR for more details https://github.com/Malax/inventory/pull/3